jplag/src/main/java/de/jplag/options/JPlagOptions.java

```diff
@@ -68,6 +70,11 @@
      * Name of the file that contains the names of files to exclude from comparison.
      */
     private String exclusionFileName;
+    
+    /**
+     * Names of the excluded files.
+     */
+    private Set<String> excludedFiles = Collections.emptySet();
```

Comment:
Is it necessary to store more thing in the options ?

At first read, it looked like JPlagOptions is the command line parsing artefact, but it is written multiple times.

In this case, excluded files are already in `JPlag`, and directly passed into `SubmissionSetBuilder`, I don't see the need to add it.

Reply:
See #192 and #274. We need the excluded files for the new report generation. Passing it through the core algorithms just to add it to the result object makes no sense. Passing it through the options is not ideal, but will do for now, as we already modify the options anyways.
If you have ideas regarding that, feel free to add it to #274.